### PR TITLE
V3 changes

### PR DESCRIPTION
**Pull request by glsoda-zz**
updating C# wrapper to match API V3 changes

### CODE REVIEW

File: Shippo/Shippo.csproj [hunk 1]
```diff
@@ -77,6 +78,9 @@
     <Compile Include="TrackingStatus.cs" />
     <Compile Include="ShippoEnums.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <None Include="packages.config" />
```
Comment (jainankit) [inline Shippo/Shippo.csproj]: can you move this back to it's original position?

File: ShippoTesting/BatchTest.cs [hunk 2]
```diff
@@ -104,15 +113,15 @@ public void TestValidPurchase ()
...
+            Assert.AreEqual (ShippoEnums.Statuses.PURCHASING, purchase.Status);
         }
 
-        [Test ()]
-        [ExpectedException (typeof (ShippoException))]
-        public void TestInvalidPurchase ()
-        {
-            getAPIResource ().PurchaseBatch ("INVALID_ID");
-        }
+        //[Test ()]
```
Comment (jainankit) [inline ShippoTesting/BatchTest.cs]: why is this commented out?

File: ShippoTesting/ManifestTest.cs [hunk 3]
```diff
@@ -10,8 +10,7 @@ public class ManifestTest : ShippoTest {
         [Test ()]
         public void TestValidCreate ()
         {
-            Manifest testObject = ManifestTest.getDefaultObject ();
-            Assert.AreEqual ("NOTRANSACTIONS", testObject.ObjectStatus);
+			Assert.That (() => ManifestTest.getDefaultObject (),Throws.TypeOf<ShippoException> ());
```
Comment (jainankit) [inline ShippoTesting/ManifestTest.cs]: this behavior changed?

File: ShippoTesting/TrackTest.cs [hunk 4]
```diff
@@ -24,12 +24,12 @@ public void TestValidGetStatus ()
             Assert.IsNotNull (track.TrackingHistory);
         }
 
-        [Test ()]
-        [ExpectedException (typeof (ShippoException))]
-        public void TestInvalidGetStatus ()
-        {
-            getAPIResource ().RetrieveTracking (CARRIER, "INVALID_ID");
-        }
+        //[Test ()]
```
Comment (jainankit) [inline ShippoTesting/TrackTest.cs]: why removed?

File: ShippoTesting/ManifestTest.cs (hunk 3 above)
Comment (glsoda-zz) [reply ShippoTesting/ManifestTest.cs]: Yes, I think we used to return a Manifest with that object status, but now we return a 400

File: Shippo/Shippo.csproj (hunk 1 above)
Comment (glsoda-zz) [reply Shippo/Shippo.csproj]: Done

File: ShippoTesting/TrackTest.cs (hunk 4 above)
Comment (glsoda-zz) [reply ShippoTesting/TrackTest.cs]: The exception testing was out of date so I had commented it but I then forgot to put it back. I've updated this as well as the other removed test in batch test

**Review by jainankit — APPROVED** [review]

**Pull request merged**